api/src/utils/common-challenge-functions.ts

```diff
@@ -12,12 +13,12 @@ const jsCertProjectIds = [
 ];
 
 const multifileCertProjectIds = getChallenges()
-  .filter(challenge => challenge.challengeType === 14)
-  .map(challenge => challenge.id);
+  .filter(c => c.challengeType === challengeTypes.multifileCertProject)
+  .map(c => c.id);
 
 const savableChallenges = getChallenges()
-  .filter(challenge => challenge.challengeType === 14)
-  .map(challenge => challenge.id);
+  .filter(c => c.challengeType === challengeTypes.multifileCertProject)
+  .map(c => c.id);
```

Comment:
```suggestion
  .filter(challenge => challenge.challengeType === challengeTypes.multifileCertProject)
  .map(challenge => challenge.id);
```

Reply:
```suggestion
  .filter(chal => chal.challengeType === challengeTypes.multifileCertProject)
  .map(chal => chal.id);
```
How about this? It's obviously a challenge and fits on one line, so does what both of us wanted :)

Reply:
go for it 👍